test/sequential/test-debugger-debug-brk.js

```diff
@@ -5,6 +5,12 @@ var spawn = require('child_process').spawn;
 
 var script = common.fixturesDir + '/empty.js';
 
+if (common.isChakraEngine) {
+  console.log('1..0 # Skipped: This test is disabled for chakra engine ' +
+  'because debugger support is not implemented yet.');
```

Comment:
This should be aligned with the string from the previous line, right? Does the linter allow this?